### PR TITLE
feat: basic navigation page

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
This PR adds organization navigation page.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Welcome to Superface
+
+Superface is a language and protocol for abstracting integrations to application use cases. It allows use case discovery and distribution of integration code at runtime.
+This approach gives you a framework to decouple lifecycle of your application and integrations it uses.
+
+
+## Technology behind
+
+<!-- One or two sentences then leading to superface.ai -->
+Superface technology consists of two parts: Registry which allows use case discovery and distribution of integration code at runtime and Client executing the integration code in your application. You can read more about our technology [here](https://superface.ai). For more details about Superface visit [how it works](https://superface.ai/how-it-works), [get started](https://superface.ai/docs/getting-started) or complete [documentation](https://superface.ai/docs).
```
Comment (freaz) [inline profile/README.md:23]: I think we should keep just one tutorial like page where to go. And not sure about getting started link and section Get started bellow.

```suggestion
Superface technology consists of two parts: Registry which allows use case discovery and distribution of integration code at runtime and Client executing the integration code in your application. You can read more about our technology [on Superface site](https://superface.ai) or to complete [documentation](https://superface.ai/docs).
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+_API Integrations the right way_
+
+[![Discord](https://img.shields.io/discord/819563244418105354?logo=discord&logoColor=fff)](https://sfc.is/discord)
+[![NPM](https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Ftwitter.com%2Fsuperfaceai)](https://twitter.com/superfaceai)
```
Comment (jnv) [inline profile/README.md:4]: This image is confusing

- it states "Tweet" while leading me to the Twitter profile
- alt for it states "NPM"
- the style doesn't match with the Discord one

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Welcome to Superface
+
+API integrations take forever to build. They’re brittle and hard to test. APIs can change on you without notice and break your integrations.
+
+Superface is trying to change this. If your Node.js app depends on multiple API integrations, Superface is right tool for you!
```
Comment (jnv) [inline profile/README.md:10]: trying to be more assertive? ;)

```suggestion
Superface is changing this. If your Node.js app depends on multiple API integrations, Superface is the right tool for you!
```

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Our motivation behind Superface is nicely described in this [video](https://www.youtube.com/watch?v=BCvq3NXFb94) from APIdays conference.
```
Comment (jnv) [inline profile/README.md:12]: Not sure if this is the best thing to link here. I'd just drop this paragraph.

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+## How it works
+
+Superface is a language and protocol for abstracting integrations to application use cases. It allows use case discovery and distribution of integration code at runtime.
```
Comment (jnv) [inline profile/README.md:16]: Just little fixes (personally I find `use-case` more readable over `use case` but didn't want to break your style):

```suggestion
Superface is a language and a protocol for abstracting API integrations to application use cases. It provides use cases discovery and distribution of integration code at runtime.
```

Here's a bit different wording with less fancy terms, does it still make sense? Feel free to pick whatever you like here.

```suggestion
Superface is a language and a protocol for turning API integrations into use cases for your business needs. With Superface you can easily discover and reuse existing use cases and update your integration code at runtime.
```

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+This approach gives you a framework to decouple lifecycle of your application and integrations it uses.
+
+Superface's technology consists of two parts:
+
+- **Registry** which allows use case discovery and distribution of integration code at runtime
```
Comment (jnv) [inline profile/README.md:22]:
```suggestion
- **Registry** for use case discovery and distribution of integration code at runtime
```

File: profile/README.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+- **Client** executing the integration code in your application
```
Comment (jnv) [inline profile/README.md:23]:
```suggestion
- **Client** for executing the integration code in your application
```

File: profile/README.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+You can read more about the technology [on Superface's site](https://superface.ai) or check the [documentation](https://superface.ai/docs).
```
Comment (jnv) [inline profile/README.md:25]:
```suggestion
You can read more about the technology [on Superface's site](https://superface.ai) and the [documentation](https://superface.ai/docs).
```

File: profile/README.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+## Get started
+
+Superface's technology can be implemented in any language, though currently we provide reference implementation for Node.js. For more details check [OneSDK repository](https://github.com/superfaceai/one-sdk-js).
```
Comment (jnv) [inline profile/README.md:29]:
```suggestion
Superface's technology can be implemented in any language. Currently we provide reference implementation for Node.js. For more details check [OneSDK repository](https://github.com/superfaceai/one-sdk-js).
```

File: profile/README.md (hunk 5 above)
Comment (freaz) [reply profile/README.md:16]: Sorry I will take only grammatical changes.

File: profile/README.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
+_Hexagonal architecture for API integrations_
+
+[![NPM](https://img.shields.io/badge/NPM-%40superfaceai-4338CA)](https://www.npmjs.com/org/superfaceai)
+[![Twitter](https://img.shields.io/badge/Twitter-%40superfaceai-4338CA)](https://twitter.com/superfaceai)
+[![Discord](https://img.shields.io/badge/Discord-%40superface.ai-4338CA)](https://sfc.is/discord)
```
Comment (freaz) [inline profile/README.md:5]: @tjerabek Can you help me with right color please? 😇 

image

**Review by zdne — APPROVED** [review]
👍🏻  minor: improve the banner please

File: profile/README.md (hunk 10 above)
Comment (freaz) [reply profile/README.md:5]: @jnv what is convention for Discord? Should it be with `@`?

Comment (jnv) [reply profile/README.md:5]: @freaz No, server names are not unique and we are not boosted for vanity name. So drop the @ please.

File: profile/README.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+_Hexagonal architecture for API integrations_
+
+[![NPM](https://img.shields.io/badge/NPM-%40superfaceai-4338CA)](https://www.npmjs.com/org/superfaceai)
+[![Twitter](https://img.shields.io/badge/Twitter-%40superfaceai-4338CA)](https://twitter.com/superfaceai)
+[![Discord](https://img.shields.io/badge/Discord-superface.ai-4338CA)](https://sfc.is/discord)
```
Comment (jnv) [inline profile/README.md:5]: A few nitpicks:

- npm is officially all lowercase, it's a recursive bacronymic abbreviation for 'npm is not an acronym'
- I think our complementary purple color is `#5850ec` (cc @tjerabek to confirm)
- expanded alt

[npm packages](https://www.npmjs.com/org/superfaceai) [Twitter profile](https://twitter.com/superfaceai) [Discord server](https://sfc.is/discord)

```suggestion
[![npm packages](https://img.shields.io/badge/npm-%40superfaceai-5850ec)](https://www.npmjs.com/org/superfaceai)
[![Twitter profile](https://img.shields.io/badge/Twitter-%40superfaceai-5850ec)](https://twitter.com/superfaceai)
[![Discord server](https://img.shields.io/badge/Discord-superface.ai-5850ec)](https://sfc.is/discord)
```

Comment (freaz) [reply profile/README.md:5]: @jnv color was taken from web. But Tom will come with colors.

Why the longer names?

Comment (jnv) [reply profile/README.md:5]: Accessibility and usability. If the badges fail to load for whatever reason, you want to have descriptive alternative text.

**Review by zdne — APPROVED** [review]

**Pull request merged**